must-gather/collection-scripts/gather_nodes

```diff
@@ -113,14 +113,19 @@ wait
 
 # Collect journal logs for specified units for all nodes
 NODE_UNITS=(kubelet crio)
-for NODE in $(oc get nodes --no-headers -o custom-columns=':metadata.name'); do
+NODES=$(oc get nodes --no-headers -o custom-columns=':metadata.name')
```

Comment:
I don't understand the reason for this change.  It looks like it is functionally the same.  It only creates a new variable that as far as I can tell is not used anywhere else.  Am I missing something?